packages/cardhost/app/helpers/scope-css.js

```diff
@@ -0,0 +1,14 @@
+import { helper } from '@ember/component/helper';
+import scope from 'scope-css';
+
+// Prefixes all CSS rules with the parent class name
+// Requires two params, the CSS and a class name or ID to add to the front of every rule.
+
+export default helper(function scopeCss(params) {
+  if (!params[0] || !params[1]) {
```

Comment:
In this case, should we throw an error? Wondering if we should be strict about making sure parent selectors are passed in so that styles are scoped.

Reply:
I was thinking that this shouldn't break our code if something doesn't have an ID yet, so I prefer the graceful failure of just not applying any CSS.